### ISSUE TITLE
Job?

### ISSUE THREAD

**Issue by jesse-schein** [slinksoft/WeeklyPubSub#1]
Do you need an actual job? Publix Super Markets is hiring lots of software developers right now, my team has openings. Reply if interested.

**Comment by slinksoft** [comment]
Yes, I am definitely interested! I currently work as a Grocery Clerk down here in South Florida and am planning to pursue a career in Information Systems up in Lakeland! I am graduating with my B.S. in Computer Science from FIU in early December. I've been at Publix for 6 years and wish to continue advancing within the company by utilizing my degree and skill set.

So far, I've applied to the entry-level Programmer position and an SQA Test Analyst position via Publix Current Openings website (http://corporate.publix.com/careers/support-areas/current-openings). I applied a week and a half ago.

Thank you for reaching out to me! I really do appreciate it!
